core/server/worker/src/main/java/alluxio/worker/block/management/tier/SwapRestoreTask.java

```diff
@@ -139,21 +139,18 @@ private Pair<List<Long>, List<BlockTransferInfo>> getSwapRestorePlan() {
           .getIterator(BlockStoreLocation.anyDirInTier(tierAlias), BlockOrder.NATURAL);
       while (tierIterator.hasNext() && moveOutBytes > 0) {
         long blockId = tierIterator.next();
-        try {
-          BlockMeta nextBlockFromTier = mEvictorView.getBlockMeta(blockId);
-          if (nextBlockFromTier == null) {
-            LOG.debug("Block:{} exist but not available for moving.", blockId);
-            continue;
-          }
-          moveOutBytes -= nextBlockFromTier.getBlockSize();
-          if (lastTier) {
-            blocksToRemove.add(nextBlockFromTier.getBlockId());
-          } else {
-            blocksToTransfer.add(BlockTransferInfo.createMove(nextBlockFromTier.getBlockLocation(),
-                nextBlockFromTier.getBlockId(), destLocation));
-          }
-        } catch (BlockDoesNotExistException e) {
-          LOG.warn("Failed to find block:{} during cascading calculation.", blockId);
+        Optional<BlockMeta> nextBlockFromTier = mEvictorView.getBlockMeta(blockId);
+        if (!nextBlockFromTier.isPresent()) {
+          LOG.debug("Block:{} exist but not available for moving.", blockId);
+          continue;
+        }
+        moveOutBytes -= nextBlockFromTier.get().getBlockSize();
+        if (lastTier) {
+          blocksToRemove.add(nextBlockFromTier.get().getBlockId());
+        } else {
+          blocksToTransfer.add(BlockTransferInfo.createMove(
+              nextBlockFromTier.get().getBlockLocation(),
+              nextBlockFromTier.get().getBlockId(), destLocation));
         }
```

Comment:
nit: I'm not a big fan of using `continue`, I think we could just use if/else to implement the control flow. what do you think?
```suggestion
        if (nextBlockFromTier.isPresent()) {
          moveOutBytes -= nextBlockFromTier.get().getBlockSize();
          if (lastTier) {
            blocksToRemove.add(nextBlockFromTier.get().getBlockId());
          } else {
            blocksToTransfer.add(BlockTransferInfo.createMove(
                nextBlockFromTier.get().getBlockLocation(),
                nextBlockFromTier.get().getBlockId(), destLocation));
          }
       } else {
          LOG.debug("Block:{} exist but not available for moving.", blockId);
       }
```

Reply:
I personally think early `continue` and early `return` works the same way in terms of code-readability and I prefer it to if / else block because it saves an indentation and removes the trouble to find a the matching "}" when a code block gets too long or have nested if / else. I found it easier to read this way. But it's a personal preference thing. I don't have strong opinions. If you strongly preference I can change it.